ExportSubscriber/Program.cs

```diff
@@ -17,7 +17,7 @@ internal static async Task Main()
             await job.Start();
 
             Console.WriteLine("Press any key to exit");
-            await Task.Run(() => { Console.ReadKey(); });
```

Comment:
Why the change was needed? Application was failing when Console.ReadKey() and suggested change to Console.Read()